tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -0,0 +1,227 @@
+<tool id="sistr_cmd" name="sistr_cmd" version="0.3.1">
+  <description>
+    Salmonella In Silico Typing Resource commandline tool for serovar prediction
+  </description>
+  <requirements>
+    <requirement type="package" version="0.3.1">sistr_cmd</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command><![CDATA[
+  sistr 
+    -f $output_format
+    #if $output_format == "csv"
+      -o sistr-report.csv
+    #elif $output_format == "json"
+      -o sistr-report.json
+    #end if
+    -p $cgmlst_profiles
+    -n $novel_alleles
+    -a $alleles_output
+    -T $tmp_dir
+    #if $keep_tmp
+      -K
+    #end if
+    #if $use_full_cgmlst_db
+      --use-full-cgmlst-db
+    #end if
+    #if $no_cgmlst
+      --no-cgmlst
+    #end if
+    #if $run_mash
+      -m
+    #end if
+    #if $qc
+      --qc
+    #end if
+    #if $threads > 1
```

Comment:
I've changed it to use ``$GALAXY_SLOTS`` in d3e07e3, but is it okay that internally ``sistr_cmd`` creates a multiprocessing threadpool if it will be run on a cluster?

https://github.com/peterk87/sistr_cmd/blob/master/sistr/sistr_cmd.py#L321